### ISSUE TITLE
Library list does not update in 0.50.7.0

### ISSUE THREAD

**Issue by Teaniel** [cadon/ARKStatsExtractor#1295]
Hi, 
Since the latest download, when I import a dino, from the extractor 'add to library' button, the library list is not updated. To refresh it, I must save and reopen (from recent files).
I wanted also thank you for the excellent work you have done on this tool!
thx
Regards
Teaniel

**Comment by Craby4GitHub** [comment]
I am noticing the same issue, here are the steps I take when adding a dino

1. In game:  Export Dino from the radial menu
2. Switch to the program and click **Import Exported Data** on the top toolbar
3. In the Exported Creatures Window: Click **Copy Values**
4. In the main program window in the Extractor Tab, give Dino a name and click **Add new to Library**
5. No dino appears in the library

I attempted to reimport the same Dino(a few different ones), but the **Exported Creatures Window** states: Already imported on X:XXam - Date

I checked both the All and specific Dino species tabs in the library.

**Comment by cadon** [comment]
Thanks for the details. It still works for me when I do it like you described.
Does it also not work in a new library? Did you check if creatures are filtered out in the `Filter library…` menu? Does the total number of creatures in the status bar at the bottom change if you add a creature that is not shown?

**Comment by Teaniel** [comment]
Hi, 
I've tested so you said, in a new library, the dino does not appear, after I click Copy values.
To make it appear I need to save file and reload file.
To be more accurate, I use W7 and my language is French.
To be noticed, my version is 0.7.50.0, and the Check for Update option returns me that my program is up to date.
I still use my first library (about 750 dinos inside), and when I import dinos, I can wait for the end of the importations before I reload file. The workaround functions everytime. 
I my opinion, we have a little refresh problem between the base in memory and the listbox where the library is viewed.

**Comment by Craby4GitHub** [comment]
> Did you check if creatures are filtered out in the `Filter library…` menu? 
Yes, it looks like they are being filtered out! I use a filter to only show Available. 

Tested again and when importing another Dino, I made sure the status is set to Available before import but when the filter is set, the dino does not appear. I also manually set the status for the Dino in the library post import but I have the same result. I have also tried these steps in a new library with the same results.
